### ISSUE TITLE
How to disable debugging logs

### ISSUE THREAD

**Issue by webmaster128** [LedgerHQ/ledgerjs#204]
Heyho!

I'm using TransportNodeHid from @ledgerhq/hw-transport-node-hid 4.21.0 in node 8 on Linux and everything works fine. But how do I get rid of those `dev 413c / 2003 : usage_page:a8a8, usage:44c5` logs written to stdout?

E.g.

```
dev 413c / 2003 : usage_page:a8a8, usage:44c5
dev 2c97 / 1 : usage_page:2e31, usage:2f32
dev 46d / c24e : usage_page:2e34, usage:2f31
dev 46d / c24e : usage_page:2d32, usage:2e34
dev 413c / 2003 : usage_page:30, usage:0
dev 2c97 / 1 : usage_page:2e31, usage:2f32
dev 46d / c24e : usage_page:2e34, usage:2f31
dev 46d / c24e : usage_page:2d32, usage:2e34
dev 413c / 2003 : usage_page:30, usage:0
dev 2c97 / 1 : usage_page:2e31, usage:2f32
dev 46d / c24e : usage_page:2e34, usage:2f31
dev 46d / c24e : usage_page:2e34, usage:2f32
dev 413c / 2003 : usage_page:30, usage:0
dev 2c97 / 1 : usage_page:2e31, usage:2f32
dev 46d / c24e : usage_page:2e34, usage:2f31
dev 46d / c24e : usage_page:2e34, usage:2f32
Transport debug: =>0101050000000580ca80000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
Transport debug: <=0101050000000a00cafe000000010090000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
```

I already set a custom debugging function to the TransportNodeHid instance but it turns out (last two lines) this only affects some other messages.

Any idea?

**Comment by gre** [comment]
this is because of node-usb

```
require('usb'). setDebugLevel(level)
```

level between 0 and 4 , not sure exactly which one disable it https://github.com/tessel/node-usb#usbsetdebuglevellevel--int

**Comment by webmaster128** [comment]
Thanks @gre!

But the dependency on the "usb" package came with @ledgerhq/hw-transport-node-hid 4.22.0, right? I am using 4.21.0 [because of this bug](https://github.com/tessel/node-usb/issues/194) in node-usb. I checked my yarn.lock again and there is no dependency on usb.

**Comment by webmaster128** [comment]
With the help of you hint I found the source:

Those debug messages come from libusb. libusb is statically linked into node-hid and node-hid provides prebuilds for my system. This meany my own system's libusb is not used. The libusb comes from ["testing VMs"](https://github.com/node-hid/node-hid/commit/00140f21556b234780cf9b86faa6da5e94acee22). If this libusb was [compiled with --enable-debug-log](https://stackoverflow.com/a/10250209), then the user has no chance to deactivate it.

I'll close this in favor of a node-hid issue as soon as I composed one.
